docs/reference/services.rst

```diff
@@ -0,0 +1,71 @@
+=================
+MicroOVN services
+=================
+
+This page presents a list of all MicroOVN services. The descriptions presented
```

Comment:
apologies, my fault here:

s/The descriptions presented here is/Their descriptions are/